.travis.yml

```diff
@@ -17,7 +17,7 @@ before_install:
       libstdc++6 libtool libgif-dev libtiff-dev libgtk2.0-dev libglib2.0-dev
       libx11-dev libxcursor-dev libxerces-c-dev libxft-dev libxinerama-dev
       libxml2-dev libxml2-utils libxmu-dev libxrandr-dev libyaml-cpp-dev libpcre3-dev
-    openssl shunit2 git git-lfs libqt4-dev qt4-default libqt4-opengl-dev
```

Comment:
I don't think it's really possible to avoid installing the system ssl libraries because so many other packages reference them.

Reply:
The system libraries should be installed by default. This is just indicating that we don't require openssl for Open GEE to run.